README.md

```diff
@@ -4,7 +4,16 @@
 
 ## 1. Create a new project
 
-1. The simplest way how to do it is to click on the green button **Use this template** on the GitHub [profile](https://github.com/seed-rs/seed-quickstart) of this quickstart.
+1. You can use cargo generate to use this template.
+
+[Learn more about `cargo generate` here.](https://github.com/ashleygwilliams/cargo-generate)
+
+```
+cargo generate --git https://github.com/seed-rs/seed-quickstart.git --name my-project
+cd my-project
+```
+
+1. Alternatively, simply click on the green button **Use this template** on the GitHub [profile](https://github.com/seed-rs/seed-quickstart) of this quickstart.

```

Comment:
![image](https://user-images.githubusercontent.com/18517402/89712661-923ac880-d992-11ea-8c66-4b6077873d76.png)

Please 
1. Fix ordering (probably by increasing indentation of the first step content)
1. Remove the unnecessary `Learn more..`.
1. Add link to `cargo generate`.
1. Add "prefix" `$` to the code and maybe mark the code block as a `bash` language.

I hope the image is explanatory enough 😃

Reply:
thanks, excellent picture, very clear 😄